tasks/system_probe.py

```diff
@@ -205,6 +205,12 @@ def build_object_files(ctx, install=True):
     set install to False to disable replacing the assets
     """
 
+    # if clang is missing, subsequent calls to ctx.run("clang ...") will fail silently, and result in us not building a
+    # new .o file
+    print("checking for clang executable...")
+    ctx.run("which clang")
+    print("found clang")
```

Comment:
Are the prints meant to be here?

Reply:
Yes,  the output for failure is a little bit confusing if you leave them out.  Because all that gets printed is a`which clang`